packages/react-table/build-single-packages.json

```diff
@@ -0,0 +1,21 @@
+{
+    "packageName": "@patternfly/react-table",
+    "modules": [ 
+        {
+            "module": "components",
+            "moduleType": "esm"
```

Comment:
We know these are modules because the array of objects is called "modules", so these can be simplified to "name" and "type".

Reply:
done